config/initializers/extends/permission_extends.rb

```diff
@@ -12,6 +12,7 @@ module ProjectPermissions
       CREATE_COMMENTS
       MANAGE_COMMENTS
       MANAGE_TAGS
+      MANAGE_ACCESS
```

Comment:
Do we actually need it alongside with MANAGE permission?

Reply:
@okriuchykhin this was decided by @radiokills to make that permissions more granular